middleware/modelExport.js

```diff
@@ -86,21 +153,14 @@ var modelExportHelpers = function modelExportHelpers (req, res) {
                     record = doc.toObject({virtuals: true });
 
                 if (count !== 0) {
-                    return getRecordData(fields, record);
+                    return getRecordData(fields, record, form, count);
                 }
 
-                var arr = [];
-
-                // get field labels
-                fields.forEach(function (fieldName) {
-                    arr.push(form[fieldName].label);
-                });
-
-                str = arr.join() + '\n';
+                let recordData = getRecordData(fields, record, form, count);
```

Comment:
Is there any requirement to create a variable? Can we just use `getRecordData` inline as it was previously?